app/controllers/users_controller.rb

```diff
@@ -0,0 +1,56 @@
+class UsersController < ApplicationController
+
+  get '/users' do
```

Comment:
do we need this route? @johnsontaiwo are we going to be needing a list of users.